src/forms/labels/input-label.js

```diff
@@ -55,34 +57,50 @@ const propTypes = {
   label: PropTypes.oneOfType([ PropTypes.string, PropTypes.bool ]),
   name: PropTypes.string.isRequired,
   tooltip: PropTypes.string,
+  required: PropTypes.bool,
+  requiredIndicator: PropTypes.oneOfType([ PropTypes.string, PropTypes.bool ]),
   ...togglePropTypes('tooltipShown')
 }
 
 const defaultProps = {
   hint: '',
   label: '',
   tooltip: '',
+  requiredIndicator: '*'
 }
 
-function InputLabel ({ hint, label, name, tooltip, tooltipShown, toggleTooltipShown }) {
+function InputLabel ({
+  hint,
+  label,
+  name,
+  tooltip,
+  tooltipShown,
+  toggleTooltipShown,
+  required,
+  requiredIndicator,
+}) {
   const labelText = label || convertNameToLabel(name)
   return (
     <span>
-      {  
+      {
         label !== false &&
         <label htmlFor={ name }>
           { labelText }
-          { 
+          {
+            required && requiredIndicator &&
+            <span aria-hidden="true">{ requiredIndicator }</span>
```

Comment:
Spacing might be an issue here so it will probably help designers to give this a class name (similar to how tooltip has one)

Reply:
great idea! do we communicate that these classnames are available somewhere in the docs?

Reply:
Hmm, I can't recall an example of that. Thoughts @dpikt ?

Reply:
We don't usually do that, unless they're dynamic. It's generally fastest to just inspect the element when styling.